### ISSUE TITLE
the question about the motivation fig

### ISSUE THREAD

**Issue by bobzhang208** [RUC-NLPIR/ARPO#31]
Thank you for your excellent work on this paper! I found the motivation experiment in Figure 2 particularly insightful.

I was wondering if you have any plans to release the code or scripts related to the experiments shown in Figure 2? It would be really helpful for understanding the setup and reproducing the results.

Looking forward to your response. Thanks again for sharing your work!

**Comment by bobzhang208** [comment]
thanks for your reply, is that means you have record the logits from the vllm source code, I have found that the OpenAI format API can only return at most 20 logprobs per token
